src/mem/ruby/Kconfig

```diff
@@ -28,15 +28,28 @@ menu "Ruby"
         bool "Enable"
 
     if RUBY
+        menu "Ruby protocols"
+
+        config USE_MULTIPLE_PROTOCOLS
+            bool 'Use multiple protocols'
+            default y
```

Comment:
Is this overridden by build_opts? I don't quite understand the interaction between build_opts / Kconfig.

If not, do we want to make default n until multiple protocols is better tested?